server/pbench/bin/pbench-report-status

```diff
@@ -70,15 +84,12 @@ if [ ! -z "$report_using_mail" -a "$report_using_mail" != "no" ] ;then
     exit $status
 fi
 
-CONFIG=/opt/pbench-server/lib/config/pbench-index.cfg
-
-server=$(getconf.py server Server)
-datestamp=${timestamp%-*}
-index=$(getconf.py index_prefix Settings)."$name"."$datestamp"
+server=$(getconf.py --config $IDXCONFIG server Server)
+# Snip off the time part of the timestamp
+datestamp=${timestamp%T*}
+index=$(getconf.py --config $IDXCONFIG index_prefix Settings)."$name"."$datestamp"
```

Comment:
Before we build the index name, can we check that we get an index prefix?

Reply:
OK